### PR TITLE
Export messed up for cubes with HS and Pruning

### PR DESCRIPTION
**Pull request by kwypchlo**
https://www.pivotaltracker.com/story/show/155080499

### CODE REVIEW

File: src/cr/cube/crunch_cube.py [hunk 1]
```diff
@@ -752,11 +777,21 @@ def labels(self, include_missing=False, include_transforms_for_dims=False):
         Returns
             labels (list of lists): Labels for each dimension
         '''
-        return [
+        labels = [
             dim.labels(include_missing, include_transforms_for_dims)
             for dim in self.dimensions
         ]
 
+        if prune:
```
Comment (slobodan-ilic) [inline src/cr/cube/crunch_cube.py]: Could you extract all that's under this `if` in a separate method, and possibly call it `_prune_labels`?

**Comment by slobodan-ilic** [comment]
I made a comment about labels pruning, and then saw that it's refactored. Nothing to add, lgtm.

**Comment by kwypchlo** [comment]
@percious Coverage increased (+0.005%) to 99.842% when pulling 4792eb8 on hs-pruning-155080499 into c4e7ff9 on master. :)

**Comment by kwypchlo** [comment]
@percious I have not added any new logic/functionalities so I relied on the old tests to cover the changes. If the code was covered and was passing before, and the same tests pass now too, it means everything is tested.

**Pull request merged**